my/time/tz/via_location.py

```diff
@@ -41,17 +41,23 @@ class config(user_config):
     # if the accuracy for the location is more than 5km, don't use
     require_accuracy: float = 5_000
 
+    # how often (hours) to refresh the cachew timezone cache
+    # this may be removed in the future if we opt for dict-based caching
```

Comment:
or maybe we'll have support for cache expiry in cachew -- wouldn't be too hard to add, and kinda useful anyway